src/Symfony/Component/Process/Process.php

```diff
@@ -180,6 +180,9 @@ public function __construct($command, string $cwd = null, array $env = null, $in
      *   $process = Process::fromShellCommandline('my_command "$MY_VAR"');
      *   $process->run(null, ['MY_VAR' => $theValue]);
      *
+     * It is also recommended to use double quotes around placeholders. This will help ensure the value
```

Comment:
this is already strongly recommended just above. Feel free to improve the above sentence if you want, but recommending twice looks duplicate to me.

but note that the comment is wrong: since https://github.com/symfony/symfony/pull/34848, ppl should use `"${:MY_VAR}"` and not `"$MY_VAR"`.

Reply:
Ok, using `"${:MY_VAR}"` should address my concern. If the developer leaves off the double quotes, the substitution won't work, so they'll be less likely to create an argument injection vulnerability. I've updated the PR to use the `"${:MY_VAR}"` syntax in the phpdoc here.